src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -249,6 +249,21 @@ public function forceFill(array $attributes)
         });
     }
 
+    /**
+     * Qualify the given column name by the model's table.
+     *
+     * @param  string  $column
+     * @return string
+     */
+    public function qualify($column)
+    {
+        if (Str::contains($column, '.')) {
+            return $column;
+        }
+
+        return $this->getTable().'.'.$column;
```

Comment:
Looks like this is missing quotations.

Reply:
Uh, what?

Reply:
I mean quotations of the table name and the column name. We already have this available via Grammar::wrap().

Reply:
Why would this suddenly be an issue? I mean I understand your idea, but this PR is only refactoring _existing code doing exactly what this line does_, nothing else.

I think if grammar shall be involved in this, it should be a separate PR anyway.